components/pages/Pages.stories.tsx

```diff
@@ -12,4 +12,4 @@ export default {
 export const index = () => <Home />;
 export const rules = () => <Rules />;
 export const user = () => <User />;
-export const lobbies = () => <Lobbies socket={null} />;
+export const lobbies = () => <Lobbies />;
```

Comment:
What happens when storybook renders `Lobbies` and there is no context? I suspect this will crash?

Reply:
@jamarob you're totally right, haven't tested this, acutally.

```
export const lobbies = () => (
  <SocketContextProvider>
    <Lobbies />
  </SocketContextProvider>
);
```

This fixes it :)